components/PDFViewer.tsx

```diff
@@ -183,13 +214,16 @@ function Nav({pageNumber, numPages}: {pageNumber: number, numPages: number}) {
             </div>
           </div>
           <div className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0">
-            <div className="bg-gray-900 text-white rounded-md px-3 py-2 text-sm font-medium">
+            <div className="bg-gray-900 text-white rounded-md px-3 py-2 text-sm font-medium m-1">
               <span>{pageNumber}</span>
               <span className="text-gray-400"> / {numPages}</span>
             </div>
+            <div className="bg-gray-900 text-white rounded-md px-3 py-2 text-sm font-medium m-1">
+              <button onClick={downloadFile}>Download</button>
```

Comment:
Instead of the `Download` text, can we use a logo: https://lucide.dev/icons/download

Reply:
Added the lucid Icon donwload

Reply:
I will be adding a new column in the Link table "allowDownload" that will help us to figure out if the user has decided to allow the download option for the document. This will be false by default.

Reply:
In the Prisma schema, in the `Link` table, I've created the newly added column, `allowDownload` with a `?` to prevent it from being deleted during a Prisma execution. This way, the existing column remains untouched, and a new column is created with a default value of false. This is tested with my vercel postgres database.

Steps I followed to add new column and prisma migration
1. Modified the `schema.prisma` file to include the new column `allowDownload`
2. Ran this command `npx prisma migrate diff --from-url <DATABASE_URL_PROD> --to-schema-datamodel prisma/schema.prisma --script > prisma/migrations/202310122339_NewColumnInLinkTable/migration.sql` to create `migration.sql` file.
3. Ran this command `npx prisma db execute --url <DATABASE_URL_PROD> --file prisma/migrations/202310122339_NewColumnInLinkTable/migration.sql` to push the schema changes to the database
4. Then ran `npx prisma generate` to update the prisma-client


### Screens:

### Link with Download option enabled
![image](https://github.com/mfts/papermark/assets/39481600/4896087d-784e-4059-852d-4cb022adda24)

### This is how users will see it
![image](https://github.com/mfts/papermark/assets/39481600/dafff9e0-60ea-44d4-86bd-8e4f714339ba)